dags/multipod/configs/maxtext_sweep_gce_config.py

```diff
@@ -56,16 +60,23 @@ def get_maxtext_sweep_gce_config(
 
   # Generate all combinations of sweep param configurations and create a TpuQueuedResourceTask for each one
   qr_task_list = []
-  current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
   for idx, config in enumerate(itertools.product(*sweep_params_list)):
     config_dict = {key: value for (key, value) in config}
 
     # Remove num_slices as a sweep param after combinations have been generated
     curr_num_slices = config_dict["NUM_SLICES"]
     del config_dict["NUM_SLICES"]
 
+    # Generate MaxText run name and tensorboard file location
+    with TaskGroup(group_id=f"generate_maxtext_inputs_{idx}") as tg:
+      run_name = maxtext.generate_maxtext_run_name(
+          run_name_prefix, curr_num_slices, tpu_version.value, tpu_cores, idx
+      )
+      tb_file_location = maxtext.generate_maxtext_tb_file_location(
+          run_name, base_output_directory
+      )
```

Comment:
This is great! I feel like this should be part of the general test infra rather than making it a `prepend` task, since every test will benefit from having a unique GCS bucket path per run. 

@RissyRan How would this fit into the existing infra?

Reply:
Yes, I agree. This could benefit other tests.